bad_connector_woocommerce/models/stock_picking/common.py

```diff
@@ -0,0 +1,83 @@
+import logging
+
+from odoo import api, fields, models
+
+from odoo.addons.component.core import Component
+
+_logger = logging.getLogger(__name__)
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    woo_bind_ids = fields.One2many(
+        comodel_name="woo.stock.picking.refund",
+        inverse_name="odoo_id",
+        string="WooCommerce Bindings(Stock)",
+        copy=False,
+    )
+    is_refund = fields.Boolean(string="Refund Quantity With Amount")
+    sale_woo_binding_ids = fields.One2many(related="sale_id.woo_bind_ids")
+    return_picking_id = fields.Many2one("stock.picking", string="Original Picking")
```

Comment:
IMO this has to be in default odoo. curious if it is not there.

Reply:
I forgot to remove this field during the modification. Apologies for the oversight, and I'll address it promptly.